tests/test_expval.py

```diff
@@ -30,7 +30,7 @@
 
 
 @pytest.mark.parametrize("theta, phi", list(zip(THETA, PHI)))
-@pytest.mark.parametrize("shots", [8192])
+@pytest.mark.parametrize("shots", [None])
```

Comment:
Why do these ones need to change?

Reply:
Same as above

Reply:
Actually, do you agree that the `shots` fixture is not actively used in any of the tests below? :confused: Unless it's hidden somewhere in a fixture. So we could set to any value.

In which case, ideally we would remove the paramtrize and `shots` argument. But ok for this PR if we want to just change to `None` as you have done :+1:

Reply:
Yep, removed it